### PR TITLE
Fix asserts in FURN tester for Icon and SVG

### PR DESCRIPTION
**Pull request by rurikoaraki**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [x] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes

The Icon and SVG test pages show asserts. This change prevents those components from being rendered (either on win32 or at all). I'm following up with @warren-ms on whether they should be fixed or deleted.

This PR also reenables the icon E2E tests

### Verification

CI

### Pull request checklist

This PR has considered (when applicable):
- [x] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

File: apps/fluent-tester/src/FluentTester/TestComponents/Icon/IconTest.tsx [hunk 1]
```diff
@@ -63,7 +64,7 @@ const Icons: React.FunctionComponent = () => {
             // We've seen some issues getting Font Awesome to link properly on Apple platforms in the FURN test app specifically.
             // This shouldn't be an issue in other apps, though, so keeping this icon Windows-only for now is an easy workaround.
             // When Android support comes, the platform check can be adjusted accordingly.
-            Platform.OS == 'windows' ? <Icon fontSource={fontCustomFontProps} width={100} height={100} color="purple" /> : null
+            Platform.OS === 'windows' ? <Icon fontSource={fontCustomFontProps} width={100} height={100} color="purple" /> : null
```
Comment (Saadnajmi) [inline apps/fluent-tester/src/FluentTester/TestComponents/Icon/IconTest.tsx:67]: 🙃

File: apps/fluent-tester/src/FluentTester/TestComponents/Svg/SvgTest.tsx [hunk 2]
```diff
@@ -118,6 +119,9 @@ const BundledSvgTest: React.FunctionComponent = () => {
 };
 
 const RemoteSvgTest: React.FunctionComponent = () => {
+  // Temporarily stop testing this case until it can be either more robust or removed
```
Comment (Saadnajmi) [inline apps/fluent-tester/src/FluentTester/TestComponents/Svg/SvgTest.tsx:122]: Add an issue tag?

**Review by Saadnajmi — APPROVED** [review]

File: apps/fluent-tester/src/FluentTester/TestComponents/Svg/SvgTest.tsx (hunk 2 above)
Comment (rurikoaraki) [reply apps/fluent-tester/src/FluentTester/TestComponents/Svg/SvgTest.tsx:122]: I was going to add an issue tag & merge this once I got a response from warren about what should be done about this test

Comment (rurikoaraki) [reply apps/fluent-tester/src/FluentTester/TestComponents/Svg/SvgTest.tsx]: Added the issue tag.

**Pull request merged**